templates/registration/login.html

```diff
@@ -23,11 +23,9 @@
 {% block navbar %}{% endblock %}
 {% block content-fluid %}
   <div id="login-content">
-    <div id="login-welcome-text">
+    <div class="login-welcome-text">
```

Comment:
Adapt the test for class instead of id: https://github.com/yverdon/geocity/blob/main/geomapshark/tests/test_login.py#L110-L112